app/src/modules/settings/routes/data-model/collections/collections.vue

```diff
@@ -99,28 +93,13 @@ const visibilityTree = computed(() => {
 	return tree;
 });
 
-const tableCollections = computed(() => {
-	return translate(
-		sortBy(
-			collectionsStore.collections.filter(
-				(collection) =>
-					isSystemCollection(collection.collection) === false && !!collection.meta === false && collection.schema,
-			),
-			['meta.sort', 'collection'],
-		),
-	);
-});
+const tableCollections = computed(() =>
+	translate(collectionsStore.databaseCollections.filter((collection) => !collection.meta)),
```

Comment:
I'm wondering if there's any better names for these different subsets. `configuredCollections` and `databaseCollections` were a little hard to grok out of the collections store file.

Reply:
The problem is that `databaseCollections` has been there before and is effectively public API at this point because it is part of the exposed API of the `CollectionsStore`